utkdigitalinitiatives/utk_isl_xml_forms
# Trac 1038

**JIRA Ticket**: [TRAC-1038](https://jira.lib.utk.edu/browse/TRAC-1038)

# What does this Pull Request do?

Creates an Articles form based on: https://docs.google.com/spreadsheets/d/1JOXRWpwz0nAnWitHmGyLbfHTTpiBXXXAeqfSeMAJhPo/edit#gid=0

# What's new?

Creates an Aritlces form with necessary CRUD operations based on the spreadsheet above.

# How should this be tested?

* Test the CRUD
    * Import the Form
    * Associate it with a content model
    * Apply any additional related transforms
    * Create a new record and select the newly associated form
    * Edit that record to verify proper CRUD behavior

# Additional Notes:
I've already walked through this a number of time to make everything work.  I suggest just importing and associating. Then, create a new XML record (As complex as you want).  Then submit.  Then edit.  First, does it read all your fields?  If so, change your fields and press submit.  When you edit again, are your new fields like they should be?

# Interested parties
@CanOfBees @cdeaneGit
